ui/src/components/ErrorBoundary.tsx

```diff
@@ -4,14 +4,24 @@ import { _ } from '@splunk/ui-utils/i18n';
 import Card from '@splunk/react-ui/Card';
 import WarningIcon from '@splunk/react-icons/enterprise/Warning';
 import errorCodes from '../constants/errorCodes';
+import { parseErrorMsg } from '../util/messageUtil';
 
 interface ErrorBoundaryProps {
     children: ReactElement | ReactElement[];
 }
 
 interface ErrorBoundaryState {
     errorCode: keyof typeof errorCodes | null;
-    error: null | unknown;
+    error?:
+        | {
+              response?: {
+                  data?: {
+                      messages?: { text: string }[];
+                  };
+              };
+          }
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        | any; // error do not have a specific type
```

Comment:
why not unknown?
```suggestion
        | null // sets in the beginning
        | unknown; // error do not have a specific type
```

Reply:
good catch, tbh i tried it but it displayed linter error for this line `return { errorCode: error.uccErrorCode, error };`
but it does not any longer, so maybe i did it on some earlier stage of refactor or sth. 
changed